### PR TITLE
Enhancement/dynamic filename + changed json1 naming to jsonl

### PR DESCRIPTION
**Pull request by SwiftPredator**
Resolves: #851

### CODE REVIEW

**Review by Hironsan — COMMENTED** [review]
Thank you for your work.
It seems to me that this does not work for document classification and sequence to sequence project. It occurred 500 error.

File: frontend/store/projects.js [hunk 1]
```diff
@@ -160,11 +160,11 @@ export const getters = {
       text: 'CSV'
     }
     const json = {
-      type: 'json',
+      type: 'jsonl',
```
Comment (fjen) [inline frontend/store/projects.js:163]: This changes the path in `views.py:TextDownloadAPI()` and wrongly executes `JSONPainter.paint_labels()`, thus resulting in error 500 for the respective project types.

We need a better way to differentiate multiple "jsonl" formats or stick with "json" here.

**Comment by fjen** [comment]
CSV export for document classification projects works fine.

**Comment by SwiftPredator** [comment]
Thanks for your reviews. Should work now for all project types. In order to be able to set the suffix type independent of the format, I added a suffix variable in the project.js. @Hironsan @fjen

**Pull request merged**